File: src/gatling/java/com/example/javaModels/Post.java
package com.example.javaModels;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.Accessors;

import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonIgnoreProperties({"id"})
@Data
@Builder(setterPrefix = "with")
@AllArgsConstructor
@NoArgsConstructor
@Accessors(chain = true)
public class Post {

    private int id;
    private String title;
    private String subject;
    private String body;
    private String category;
    private int user;
    private List<Integer> comments;
    private String createdAt;

    public Post addComments(Integer... commentIds){
        this.comments.addAll(Arrays.asList(commentIds));
        return this;
    }

}

File: src/gatling/kotlin/com/example/modelds/Post.kt
package com.example.modelds

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonInclude

@JsonIgnoreProperties("id")
@JsonInclude(JsonInclude.Include.NON_NULL)
data class Post(
    val id: Int? = null,
    val title: String,
    val subject: String,
    val body: String,
    val category: String,
    val user: Int,
    val comments: MutableList<Int>,
    val createdAt: String): Model {

        fun addComments(vararg ids: Int): Post {
            ids.forEach(comments::add)
            return this;
        }


    }